js-extensions/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/js-extensions",
-  "version": "0.0.14",
+  "version": "0.0.15-beta4",
```

Comment:
A lot of the tests use "sh" steps in pipeline, which I don't think will work on windows. They could be changed to simply echo output - and then work across platforms (and would serve the same purpose, as long as the assertions are still there).